TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -599,6 +599,18 @@ TYPO3:
             namespaces:
               f: 'TYPO3\Fluid\ViewHelpers'
               neos: 'TYPO3\Neos\ViewHelpers'
+      'Neos:TypoScriptViewHelpers':
```

Comment:
okay, thanks for the information!
I have no deeper knowledege how docs are generated, but the `TypoScipt.rst` file does not show up here:
https://github.com/neos/neos-development-collection/tree/master/TYPO3.Neos/Documentation/References/ViewHelpers
That's why i was wondering